cves/kernel/CVE-2017-16913.yml

```diff
@@ -151,10 +150,11 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: The vulnerability was originally discovered by Jakub Jirasek of Secunia Research
```

Comment:
How was the vulnerability discovered by Jakub?

Reply:
According the redhat bugzilla link - https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2017-16913 -, the bug was found by Secunia Research.

Using the wayback machine I found their website which shows that Jakub Jirasek found the DoS in the usb-ip subsystem - https://web.archive.org/web/20200227134938/https://www.flexera.com/products/operations/software-vulnerability-research/secunia-research/advisories.html